components/automate-cli/pkg/verifyserver/mock/mockserver.go

```diff
@@ -0,0 +1,65 @@
+package mock
```

Comment:
I think this is a helper to write unit tests for the handlers.This can be moved to the common library so that it can be used any code across the repo

Reply:
moved it to common lib/mockserver